src/components/sidenav/sidenav.spec.js

```diff
@@ -173,6 +173,85 @@ describe('mdSidenav', function() {
 
   });
 
+  describe("focus", function() {
+
+    var $material, $mdInteraction;
+
+    beforeEach( inject(function(_$material_, _$mdInteraction_) {
+      $material = _$material_;
+      $mdInteraction = _$mdInteraction_;
+    }));
+
+    function flush() {
+      $material.flushInterimElement();
+    }
+
+    function setupTrigger() {
+      var el;
+      inject(function($compile, $rootScope) {
+        var parent = angular.element(document.body);
+        el = angular.element('<button aria-label="Toggle Menu">Toggle</button>');
```

Comment:
So that the tests show correct requirements, you don't need `aria-label` here since the button has text in it.